src/sqswatcher/plugins/slurm.py

```diff
@@ -208,5 +209,50 @@ def update_cluster(max_cluster_size, cluster_user, update_events, instance_prope
     return failed, succeeded
 
 
+def _is_node_locked(hostname):
+    node_state = get_node_state(hostname)
+    log.info("Node {} currently in state {}".format(hostname, node_state))
+    for disable_state in SLURM_NODE_DISABLED_STATES:
+        if disable_state in node_state:
+            return True
+    return False
+
+
+def perform_health_actions(health_events):
+    """Update and write node lists( and gres_nodes if instance has GPU); restart relevant nodes."""
+    failed = []
+    succeeded = []
+    for event in health_events:
+        try:
+            # to-do, ignore fail to lock message if node is not in scheduler
+            if _is_node_locked(event.host.hostname):
+                log.error(
```

Comment:
why is this an error?also do we need this check or we can go ahead an lock the node?